code/game/machinery/lightswitch.dm

```diff
@@ -61,3 +65,23 @@
 		return
 	if(!(stat & (BROKEN|NOPOWER)))
 		power_change()
+
+/obj/machinery/light_switch/proc/shock(mob/user)
+	if(stat & (BROKEN|NOPOWER))		// unpowered, no shock
+		return FALSE
+	var/datum/effect_system/spark_spread/s = new /datum/effect_system/spark_spread
+	s.set_up(5, 1, src)
+	s.start()
+	if (electrocute_mob(user, get_area(src), src, 0.7, TRUE))
+		return TRUE
+	else
+		return FALSE
+
+/obj/machinery/light_switch/emag_act(mob/user)
+	if(obj_flags & EMAGGED)
+		to_chat(user, "<span class='warning'>Nothing new seems to happen when you swipe the emag.</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>Nothing seems to happen when you swipe the emag.</span>")
```

Reply:
No.
Your change doesn't make sense because in order to arrive to this message you need to already swipe it once to get an effect. This conveys that the same result is achieved with nothing "new" visible.